### PR TITLE
Converto to TS Callback and Courses component

### PR DESCRIPTION
**Pull request by DavielSA**
Can you review this issues fixed, please?

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -48,11 +48,11 @@ export class App extends React.Component<Props, State> {
           />
           <Route
             path="/callback"
-            render={(props: Props) => <Callback auth={auth} {...props} />}
+            render={(props: Props) => <Callback auth={auth} location={null} {...props} />}
```
Comment (dailymp) [inline src/App.tsx]: location must be dynamic not only null, please fix it

File: src/Auth/Auth.ts [hunk 2]
```diff
@@ -129,7 +128,7 @@ export default class Auth {
     if (delay > 0) setTimeout(() => this.renewToken(undefined), delay);
   }
 
-  private renewToken = (cb:any):void=> {
+  public renewToken = (cb:any):void=> {
```
Comment (dailymp) [inline src/Auth/Auth.ts]: could you type cb?

File: src/App.tsx (hunk 1 above)
Comment (DavielSA) [reply src/App.tsx]: OK. Done

File: src/Auth/Auth.ts (hunk 2 above)
Comment (DavielSA) [reply src/Auth/Auth.ts]: OK. Done

**Pull request merged**